front50-sql/src/main/kotlin/com/netflix/spinnaker/config/SqlConfiguration.kt

```diff
@@ -39,14 +41,35 @@ class SqlConfiguration : CommonStorageServiceDAOConfig() {
     objectMapper: ObjectMapper,
     registry: Registry,
     jooq: DSLContext,
-    sqlProperties: SqlProperties
+    sqlProperties: SqlProperties,
+    @Value("\${sql.primary.pool-name:default}") poolName: String
   ): SqlStorageService =
     SqlStorageService(
       objectMapper,
       registry,
       jooq,
       Clock.systemDefaultZone(),
       sqlProperties.retries,
-      1000
+      1000,
+      poolName
+    )
+
+  @Bean
+  @ConditionalOnProperty("sql.secondary.enabled", matchIfMissing = false)
+  fun secondarySqlStorageService(
+    objectMapper: ObjectMapper,
+    registry: Registry,
+    @Qualifier("secondaryJooq") jooq: DSLContext,
+    sqlProperties: SqlProperties,
+    @Value("\${sql.secondary.pool-name}") poolName: String
```

Comment:
I think it would be cleaner to define both `sql.primary.pool-name` and `sql.secondary.pool-name` in a class annotated with `@ConfigurationProperties`.   Informally, I think we're trying to steer clear of introducing new `@Value` annotations.

Reply:
Instead of introducing more props, derived the pool name using the existing connection pool config attributes.  This way we don't have to duplicate the name into yet another prop.